### DISCUSSION TITLE
03-Aug-20 The page build failed

### DISCUSSION THREAD

**Discussion by MSP-Greg** [community/community#22743] [Other Feature Feedback, Questions, & Ideas]
<p>Repeated failures at:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/MSP-Greg/MSP-Greg.github.io/commits/master" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/15081176/181072776-daa08c1d-db2a-4294-a150-950457fced3b.jpeg" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/MSP-Greg/MSP-Greg.github.io/commits/master" target="_blank" rel="nofollow noopener">MSP-Greg/MSP-Greg.github.io</a></h3>

<p>Ruby and other documentation created with yard-t2, tablet friendly - MSP-Greg/MSP-Greg.github.io</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Maintained the repo/site for quite a while, so I think this issue is a GitHub issue.</p>

**Comment by tcbyrd** [comment, marked as answer]
<p>Hey there! We were having some issues with builds timing out. This should be resolved now, so if you trigger a re-build it should work now. Sorry for the trouble!</p>

**Comment by MSP-Greg** [comment]
<p>Thank you.  Built a few minutes ago.</p>
